build.gradle

```diff
@@ -1,8 +1,15 @@
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2024.3.2"
-    id("com.diffplug.spotless") version "7.0.0.BETA2"
-    id 'org.jetbrains.kotlin.jvm'
+    id "edu.wpi.first.GradleRIO" version "2025.1.1-beta-3"
+    id "com.diffplug.spotless" version "7.0.0.BETA4"
```

Comment:
Sorry I should've been clearer, latest stable version. I don't think we should be shipping with a beta version

Reply:
The current version on the main branch is beta 2 of spotless; should I revert it?

Reply:
Yes idk why we shipping with beta spotless

Reply:
I would say leave it. Spotless is being annoying these days and not updating the non-beta branches. It makes it hard to use the IntelliJ plugins like ktfmt or google-java-format because the plugins support a newer version of those tools, but the latest stable version of spotless does not.